util.go

```diff
@@ -534,3 +538,14 @@ func Open(driver, dsn string) (*DB, error) {
 // handle.Wrap -> Wrap yourself
 // handle.Tx -> NewTransaction
 // handle.Prepare -> NewPrearedStatement
+
```

Comment:
Copying data seems not to be the ideal solution to the problem. Why does the data need to be addressable in the first place? For INSERT / UPDATE we only need to read the data, unless we want use the set primary key feature of the library, 
 which will not do anything useful on the copy anyways.

Reply:
I removed the check to copy the data, I definitely don't need it at update, I now only check if the field is addressable before setting it, but if I wonder if it should throw an error in case it does want to set primary key and the data was not passed by pointer and is not addressable